Gruntfile.js

```diff
@@ -1242,30 +1198,25 @@ module.exports = function(grunt) {
     // Builds Android from scratch by recreating the Cordova environment.
     'exec:cleanAndroid',
     'build_cca',
-    'exec:ccaCreateDev',
-    'exec:ccaPlatformAndroidDev',
-    'exec:ccaAddPluginsAndroidDev',
-    'copy:cca_splash_dev',
+    'exec:ccaCreate',
+    'exec:ccaPlatformAndroid',
+    'exec:ccaAddPluginsAndroid',
+    'copy:cca_splash',
     'build_android_lite'
   ]);
   registerTask(grunt, 'build_android_lite', [
     // Android build task that does not recreate the Cordova environment.
     // Should only be used for building CCA modules and after running
     // build_android, without cleaning, initially at least once.
     'build_cca',
-    'exec:ccaBuildAndroid',
+    'exec:ccaBuildAndroidDev',
     'exec:androidReplaceXwalkDev'
   ]);
   registerTask(grunt, 'release_android', [
-    'build_cca', 
-    'copy:dist', 
-    'exec:ccaCreateDist', 
-    'exec:ccaPlatformAndroidDist', 
-    'exec:ccaAddPluginsAndroidDist', 
-    'copy:cca_splash_dist', 
+    'build_android',
```

Comment:
So `release_android` was also building the debug build - good catch.

I mostly did what you suggested: renamed `build_android` to `recreate_cca_env`, created `android_debug` and `android_release` which each have a `_lite` option. No need to change `replace_xwalk_in_apk.sh`, though, AFAICT.